src/main/java/rsb/internal/globval/GlobalConfiguration.java

```diff
@@ -47,27 +47,38 @@ public static String getAccountsFile() {
 			if (GlobalConfiguration.getCurrentOperatingSystem() == OperatingSystem.WINDOWS) {
 				path = System.getenv("APPDATA") + File.separator
 						+ GlobalConfiguration.NAME + "_Accounts.ini";
+			} else if (GlobalConfiguration.getCurrentOperatingSystem() == OperatingSystem.LINUX) {
+				path = Paths.getUnixHome() + File.separator
+						+ ".config" + File.separator
+						+ GlobalConfiguration.NAME_LOWERCASE + "_acct.ini";
```

Comment:
Linux users don't like when you pollute `~` - Config files should be in `.config`

Reply:
Understandable. I don't have a Linux environment and am not heavy into it at the moment.